composer.json

```diff
@@ -21,12 +21,12 @@
     ],
     "require": {
         "php": "^5.6|^7.0",
-
-        "symfony/framework-bundle": "^2.8|^3.0"
+        "symfony/framework-bundle": "^4.3"
```

Comment:
We can probably keep support for v`3.4` here as well? IE `^3.4|^4.0`? Same for the packages below.

Reply:
Done.